microprofile/graphql/server/pom.xml

```diff
@@ -82,6 +82,14 @@
             <plugin>
                 <groupId>org.jboss.jandex</groupId>
                 <artifactId>jandex-maven-plugin</artifactId>
+                <!-- Temporal fix meanwhile there are no newer versions of the plugin -->
+                <dependencies>
+                    <dependency>
+                        <groupId>org.jboss</groupId>
+                        <artifactId>jandex</artifactId>
+                        <version>3.1.8</version>
```

Comment:
Why not upgrade jandex in dependencies/pom.xml and add this workaround to the top level pom.xml ?

Reply:
It is tricky, but I am working on that.

Originally I just wanted to bypass this punctual issue, because it is happening in test, and we already have other tests that are generating indexes and reading them fine.

Making this upgrade in the main plugin requires some major changes, because it generates indexes in a higher version and we need to update other libs related to jandex to read them with a version that is supported. I already solved many of the issues, but I am struggling with 2 tests of tck open-api.

Reply:
Increasing version does not help, but I debugged it, and the number comes as intener, so it is trunkated after 123. Adding `type = SchemaType.STRING` in the TCK fixes the problem:
`example = "123-456-7891", type = SchemaType.STRING`

I am going to explore it more, or maybe I will create a PR in the TCK and lets see what happens.

Reply:
It seems it is correctly set in io.smallrye.openapi.api.models.media.SchemaImpl#setExample in this stacktrace:
```
Thread [main] (Suspended (breakpoint at line 652 in io.smallrye.openapi.api.models.media.SchemaImpl))	
	owns: org.jboss.weld.Container  (id=155)	
	owns: org.eclipse.microprofile.openapi.tck.AirlinesAppTest  (id=156)	
	io.smallrye.openapi.api.models.media.SchemaImpl.setExample(java.lang.Object) line: 652	
	io.smallrye.openapi.runtime.io.schema.SchemaFactory.readSchema(io.smallrye.openapi.runtime.scanner.spi.AnnotationScannerContext, org.eclipse.microprofile.openapi.models.media.Schema, org.jboss.jandex.AnnotationInstance, java.util.Map<java.lang.String,java.lang.Object>) line: 183	
	io.smallrye.openapi.runtime.scanner.dataobject.AnnotationTargetProcessor.readSchemaAnnotatedField(java.lang.String, org.jboss.jandex.AnnotationInstance, org.jboss.jandex.Type) line: 337	
	io.smallrye.openapi.runtime.scanner.dataobject.AnnotationTargetProcessor.processField() line: 163	
	io.smallrye.openapi.runtime.scanner.dataobject.AnnotationTargetProcessor.process(io.smallrye.openapi.runtime.scanner.spi.AnnotationScannerContext, io.smallrye.openapi.runtime.scanner.dataobject.DataObjectDeque, io.smallrye.openapi.runtime.scanner.dataobject.TypeResolver, io.smallrye.openapi.runtime.scanner.dataobject.DataObjectDeque$PathEntry) line: 65	
	io.smallrye.openapi.runtime.scanner.OpenApiDataObjectScanner.lambda$depthFirstGraphSearch$1(io.smallrye.openapi.runtime.scanner.dataobject.DataObjectDeque$PathEntry, io.smallrye.openapi.runtime.scanner.dataobject.TypeResolver) line: 263	
	io.smallrye.openapi.runtime.scanner.OpenApiDataObjectScanner$$Lambda.0x00007fc93c5118d8.accept(java.lang.Object) line: not available	
	java.util.stream.ForEachOps$ForEachOp$OfRef<T>.accept(T) line: 184	
	java.util.stream.ReferencePipeline$2$1.accept(P_OUT) line: 179	
	java.util.LinkedHashMap$LinkedValueIterator(java.util.Iterator<E>).forEachRemaining(java.util.function.Consumer<? super E>) line: 133	
	java.util.Spliterators$IteratorSpliterator<T>.forEachRemaining(java.util.function.Consumer<? super T>) line: 1939	
	java.util.stream.ReferencePipeline$2(java.util.stream.AbstractPipeline<E_IN,E_OUT,S>).copyInto(java.util.stream.Sink<P_IN>, java.util.Spliterator<P_IN>) line: 509	
	java.util.stream.ReferencePipeline$2(java.util.stream.AbstractPipeline<E_IN,E_OUT,S>).wrapAndCopyInto(S, java.util.Spliterator<P_IN>) line: 499	
	java.util.stream.ForEachOps$ForEachOp$OfRef<T>(java.util.stream.ForEachOps$ForEachOp<T>).evaluateSequential(java.util.stream.PipelineHelper<T>, java.util.Spliterator<S>) line: 151	
	java.util.stream.ForEachOps$ForEachOp$OfRef<T>.evaluateSequential(java.util.stream.PipelineHelper, java.util.Spliterator) line: 174	
	java.util.stream.ReferencePipeline$2(java.util.stream.AbstractPipeline<E_IN,E_OUT,S>).evaluate(java.util.stream.TerminalOp<E_OUT,R>) line: 234	
	java.util.stream.ReferencePipeline$2(java.util.stream.ReferencePipeline<P_IN,P_OUT>).forEach(java.util.function.Consumer<? super P_OUT>) line: 596	
	io.smallrye.openapi.runtime.scanner.OpenApiDataObjectScanner.depthFirstGraphSearch() line: 263	
	io.smallrye.openapi.runtime.scanner.OpenApiDataObjectScanner.process() line: 213	
	io.smallrye.openapi.runtime.scanner.OpenApiDataObjectScanner.process(io.smallrye.openapi.runtime.scanner.spi.AnnotationScannerContext, org.jboss.jandex.Type) line: 159	
	io.smallrye.openapi.runtime.io.schema.SchemaFactory.introspectClassToSchema(io.smallrye.openapi.runtime.scanner.spi.AnnotationScannerContext, org.jboss.jandex.ClassType, boolean) line: 576	
	io.smallrye.openapi.runtime.io.schema.SchemaFactory.typeToSchema(io.smallrye.openapi.runtime.scanner.spi.AnnotationScannerContext, org.jboss.jandex.Type, java.util.List<io.smallrye.openapi.runtime.scanner.AnnotationScannerExtension>) line: 454	
	io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.lambda$processClassSchemas$3(io.smallrye.openapi.runtime.scanner.spi.AnnotationScannerContext, org.jboss.jandex.Type) line: 208	
	io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner$$Lambda.0x00007fc93c50c008.accept(java.lang.Object) line: not available	
	java.util.stream.ForEachOps$ForEachOp$OfRef<T>.accept(T) line: 184	
	java.util.stream.ReferencePipeline$3$1.accept(P_OUT) line: 197	
	java.util.stream.ReferencePipeline$2$1.accept(P_OUT) line: 179	
	java.util.ArrayList$ArrayListSpliterator.forEachRemaining(java.util.function.Consumer<? super E>) line: 1708	
	java.util.stream.ReferencePipeline$3(java.util.stream.AbstractPipeline<E_IN,E_OUT,S>).copyInto(java.util.stream.Sink<P_IN>, java.util.Spliterator<P_IN>) line: 509	
	java.util.stream.ReferencePipeline$3(java.util.stream.AbstractPipeline<E_IN,E_OUT,S>).wrapAndCopyInto(S, java.util.Spliterator<P_IN>) line: 499	
	java.util.stream.ForEachOps$ForEachOp$OfRef<T>(java.util.stream.ForEachOps$ForEachOp<T>).evaluateSequential(java.util.stream.PipelineHelper<T>, java.util.Spliterator<S>) line: 151	
	java.util.stream.ForEachOps$ForEachOp$OfRef<T>.evaluateSequential(java.util.stream.PipelineHelper, java.util.Spliterator) line: 174	
	java.util.stream.ReferencePipeline$3(java.util.stream.AbstractPipeline<E_IN,E_OUT,S>).evaluate(java.util.stream.TerminalOp<E_OUT,R>) line: 234	
	java.util.stream.ReferencePipeline$3(java.util.stream.ReferencePipeline<P_IN,P_OUT>).forEach(java.util.function.Consumer<? super P_OUT>) line: 596	
	io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.processClassSchemas(io.smallrye.openapi.runtime.scanner.spi.AnnotationScannerContext) line: 208	
	io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.scanMicroProfileOpenApiAnnotations() line: 153	
	io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.scan(java.lang.String...) line: 118	
	io.helidon.microprofile.openapi.MPOpenAPISupport(io.helidon.openapi.OpenAPISupport).lambda$expandModelUsingAnnotations$3(io.smallrye.openapi.api.OpenApiConfig, java.util.concurrent.atomic.AtomicReference, org.jboss.jandex.IndexView) line: 415	
	io.helidon.openapi.OpenAPISupport$$Lambda.0x00007fc93c505598.accept(java.lang.Object) line: not available	
	java.util.ArrayList<E>.forEach(java.util.function.Consumer<? super E>) line: 1596	
	io.helidon.microprofile.openapi.MPOpenAPISupport(io.helidon.openapi.OpenAPISupport).expandModelUsingAnnotations(io.smallrye.openapi.api.OpenApiConfig, java.util.List<? extends org.jboss.jandex.IndexView>) line: 412	
	io.helidon.microprofile.openapi.MPOpenAPISupport(io.helidon.openapi.OpenAPISupport).prepareModel(io.smallrye.openapi.api.OpenApiConfig, io.smallrye.openapi.runtime.OpenApiStaticFile, java.util.List<? extends org.jboss.jandex.IndexView>) line: 381	
	io.helidon.microprofile.openapi.MPOpenAPISupport(io.helidon.openapi.OpenAPISupport).lambda$model$0() line: 255	
	io.helidon.openapi.OpenAPISupport$$Lambda.0x00007fc93c4e6ad8.get() line: not available	
	io.helidon.openapi.OpenAPISupport.access(java.util.concurrent.locks.Lock, java.util.function.Supplier<T>) line: 1030	
	io.helidon.microprofile.openapi.MPOpenAPISupport(io.helidon.openapi.OpenAPISupport).model() line: 253	
	io.helidon.microprofile.openapi.MPOpenAPISupport(io.helidon.openapi.OpenAPISupport).prepareModel() line: 236	
	io.helidon.microprofile.openapi.MPOpenAPISupport.prepareModel() line: 32	
	io.helidon.microprofile.openapi.OpenApiCdiExtension.buildModel(java.lang.Object) line: 118	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c350400.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.jboss.weld.injection.VirtualMethodInjectionPoint<T,X>(org.jboss.weld.injection.StaticMethodInjectionPoint<T,X>).invoke(java.lang.Object, java.lang.Object[], java.lang.Class<? extends java.lang.RuntimeException>) line: 95	
	org.jboss.weld.injection.VirtualMethodInjectionPoint<T,X>(org.jboss.weld.injection.StaticMethodInjectionPoint<T,X>).invoke(java.lang.Object, java.lang.Object, org.jboss.weld.manager.BeanManagerImpl, jakarta.enterprise.context.spi.CreationalContext<?>, java.lang.Class<? extends java.lang.RuntimeException>) line: 85	
	org.jboss.weld.injection.MethodInvocationStrategy$SimpleMethodInvocationStrategy.invoke(java.lang.Object, org.jboss.weld.injection.MethodInjectionPoint<?,?>, T, org.jboss.weld.manager.BeanManagerImpl, jakarta.enterprise.context.spi.CreationalContext<?>) line: 168	
	org.jboss.weld.event.ExtensionObserverMethodImpl<T,X>(org.jboss.weld.event.ObserverMethodImpl<T,X>).sendEvent(T, java.lang.Object, jakarta.enterprise.context.spi.CreationalContext<?>) line: 330	
	org.jboss.weld.event.ExtensionObserverMethodImpl<T,X>.sendEvent(T, java.lang.Object, jakarta.enterprise.context.spi.CreationalContext<?>) line: 123	
	org.jboss.weld.event.ExtensionObserverMethodImpl<T,X>(org.jboss.weld.event.ObserverMethodImpl<T,X>).sendEvent(T) line: 308	
	org.jboss.weld.event.ExtensionObserverMethodImpl<T,X>(org.jboss.weld.event.ObserverMethodImpl<T,X>).notify(T) line: 286	
	org.jboss.weld.event.ExtensionObserverMethodImpl<T,X>(jakarta.enterprise.inject.spi.ObserverMethod<T>).notify(jakarta.enterprise.inject.spi.EventContext<T>) line: 125	
	org.jboss.weld.util.Observers.notify(jakarta.enterprise.inject.spi.ObserverMethod<? super T>, T, jakarta.enterprise.inject.spi.EventMetadata) line: 166	
	org.jboss.weld.event.ObserverNotifier.notifySyncObservers(java.util.List<jakarta.enterprise.inject.spi.ObserverMethod<? super T>>, T, jakarta.enterprise.inject.spi.EventMetadata, org.jboss.weld.event.ObserverNotifier.ObserverExceptionHandler) line: 285	
	org.jboss.weld.event.ObserverNotifier.notify(org.jboss.weld.event.ResolvedObservers<T>, T, jakarta.enterprise.inject.spi.EventMetadata) line: 273	
	org.jboss.weld.event.EventImpl<T>.fire(T) line: 96	
	io.helidon.microprofile.cdi.HelidonContainerImpl.doStart() line: 340	
	io.helidon.microprofile.cdi.HelidonContainerImpl$$Lambda.0x00007fc93c346f10.call() line: not available	
	io.helidon.common.context.Contexts.runInContext(io.helidon.common.context.Context, java.util.concurrent.Callable<T>) line: 137	
	io.helidon.microprofile.cdi.HelidonContainerImpl.start() line: 250	
	io.helidon.microprofile.server.ServerImpl.start() line: 76	
	io.helidon.microprofile.arquillian.ServerRunner.start(org.eclipse.microprofile.config.Config, int) line: 73	
	java.lang.invoke.LambdaForm$DMH.0x00007fc93c1a0000.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object, int) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c1a1800.invoke(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c1a1c00.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 155	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	io.helidon.microprofile.arquillian.HelidonDeployableContainer.startServer(io.helidon.microprofile.arquillian.HelidonDeployableContainer$RunContext, java.nio.file.Path[]) line: 318	
	io.helidon.microprofile.arquillian.HelidonDeployableContainer.deploy(org.jboss.shrinkwrap.api.Archive<?>) line: 196	
	org.jboss.arquillian.container.impl.client.container.ContainerDeployController$3.call() line: 151	
	org.jboss.arquillian.container.impl.client.container.ContainerDeployController$3.call() line: 118	
	org.jboss.arquillian.container.impl.client.container.ContainerDeployController.executeOperation(java.util.concurrent.Callable<java.lang.Void>) line: 239	
	org.jboss.arquillian.container.impl.client.container.ContainerDeployController.deploy(org.jboss.arquillian.container.spi.event.DeployDeployment) line: 118	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c110000.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.jboss.arquillian.core.impl.ObserverImpl.invoke(org.jboss.arquillian.core.spi.Manager, java.lang.Object) line: 86	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.invokeObservers() line: 103	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.proceed() line: 90	
	org.jboss.arquillian.container.impl.client.container.DeploymentExceptionHandler.verifyExpectedExceptionDuringDeploy(org.jboss.arquillian.core.spi.EventContext<org.jboss.arquillian.container.spi.event.DeployDeployment>) line: 47	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c110000.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.jboss.arquillian.core.impl.ObserverImpl.invoke(org.jboss.arquillian.core.spi.Manager, java.lang.Object) line: 86	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.proceed() line: 95	
	org.jboss.arquillian.container.impl.client.ContainerDeploymentContextHandler.createDeploymentContext(org.jboss.arquillian.core.spi.EventContext<org.jboss.arquillian.container.spi.event.DeploymentEvent>) line: 71	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c110000.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.jboss.arquillian.core.impl.ObserverImpl.invoke(org.jboss.arquillian.core.spi.Manager, java.lang.Object) line: 86	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.proceed() line: 95	
	org.jboss.arquillian.container.impl.client.ContainerDeploymentContextHandler.createContainerContext(org.jboss.arquillian.core.spi.EventContext<org.jboss.arquillian.container.spi.event.ContainerControlEvent>) line: 54	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c110000.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.jboss.arquillian.core.impl.ObserverImpl.invoke(org.jboss.arquillian.core.spi.Manager, java.lang.Object) line: 86	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.proceed() line: 95	
	org.jboss.arquillian.core.impl.ManagerImpl.fire(T, org.jboss.arquillian.core.spi.NonManagedObserver<T>) line: 133	
	org.jboss.arquillian.core.impl.ManagerImpl.fire(java.lang.Object) line: 105	
	org.jboss.arquillian.core.impl.EventImpl<T>.fire(T) line: 62	
	org.jboss.arquillian.container.impl.client.container.ContainerDeployController$1.perform(org.jboss.arquillian.container.spi.Container, org.jboss.arquillian.container.spi.client.deployment.Deployment) line: 92	
	org.jboss.arquillian.container.impl.client.container.ContainerDeployController$1.perform(java.lang.Object, java.lang.Object) line: 77	
	org.jboss.arquillian.container.impl.client.container.ContainerDeployController.forEachDeployment(java.util.List<org.jboss.arquillian.container.spi.client.deployment.Deployment>, org.jboss.arquillian.container.impl.client.container.ContainerDeployController.Operation<org.jboss.arquillian.container.spi.Container,org.jboss.arquillian.container.spi.client.deployment.Deployment>) line: 232	
	org.jboss.arquillian.container.impl.client.container.ContainerDeployController.forEachManagedDeployment(org.jboss.arquillian.container.impl.client.container.ContainerDeployController.Operation<org.jboss.arquillian.container.spi.Container,org.jboss.arquillian.container.spi.client.deployment.Deployment>) line: 212	
	org.jboss.arquillian.container.impl.client.container.ContainerDeployController.deployManaged(org.jboss.arquillian.container.spi.event.DeployManagedDeployments) line: 77	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c110000.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.jboss.arquillian.core.impl.ObserverImpl.invoke(org.jboss.arquillian.core.spi.Manager, java.lang.Object) line: 86	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.invokeObservers() line: 103	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.proceed() line: 90	
	org.jboss.arquillian.core.impl.ManagerImpl.fire(T, org.jboss.arquillian.core.spi.NonManagedObserver<T>) line: 133	
	org.jboss.arquillian.core.impl.ManagerImpl.fire(java.lang.Object) line: 105	
	org.jboss.arquillian.core.impl.EventImpl<T>.fire(T) line: 62	
	org.jboss.arquillian.container.test.impl.client.ContainerEventController.execute(org.jboss.arquillian.test.spi.event.suite.BeforeClass) line: 96	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c0bcc00.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.jboss.arquillian.core.impl.ObserverImpl.invoke(org.jboss.arquillian.core.spi.Manager, java.lang.Object) line: 86	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.invokeObservers() line: 103	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.proceed() line: 90	
	org.jboss.arquillian.test.impl.TestContextHandler.createClassContext(org.jboss.arquillian.core.spi.EventContext<org.jboss.arquillian.test.spi.event.suite.ClassEvent>) line: 83	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c110000.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.jboss.arquillian.core.impl.ObserverImpl.invoke(org.jboss.arquillian.core.spi.Manager, java.lang.Object) line: 86	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.proceed() line: 95	
	org.jboss.arquillian.test.impl.TestContextHandler.createSuiteContext(org.jboss.arquillian.core.spi.EventContext<org.jboss.arquillian.test.spi.event.suite.SuiteEvent>) line: 69	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c110000.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.jboss.arquillian.core.impl.ObserverImpl.invoke(org.jboss.arquillian.core.spi.Manager, java.lang.Object) line: 86	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.proceed() line: 95	
	org.jboss.arquillian.core.impl.ManagerImpl.fire(T, org.jboss.arquillian.core.spi.NonManagedObserver<T>) line: 133	
	org.jboss.arquillian.core.impl.ManagerImpl.fire(java.lang.Object) line: 105	
	org.jboss.arquillian.test.impl.EventTestRunnerAdaptor.beforeClass(java.lang.Class<?>, org.jboss.arquillian.test.spi.LifecycleMethodExecutor) line: 89	
	org.eclipse.microprofile.openapi.tck.AirlinesAppTest(org.jboss.arquillian.testng.Arquillian).arquillianBeforeClass() line: 99	
	java.lang.invoke.LambdaForm$DMH.0x00007fc93c0a8c00.invokeVirtual(java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c0a9400.invoke(java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 153	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(java.lang.reflect.Method, java.lang.Object, java.lang.Object[]) line: 139	
	org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(org.testng.ITestNGMethod, org.testng.internal.ConstructorOrMethod, java.lang.Object, java.lang.Object[], org.testng.ITestResult) line: 69	
	org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(java.lang.Object, org.testng.ITestNGMethod, java.lang.Object[], org.testng.ITestResult) line: 361	
	org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(org.testng.internal.invokers.ConfigMethodArguments) line: 296	
	org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(org.testng.ITestClass, org.testng.IMethodInstance) line: 180	
	org.testng.internal.invokers.TestMethodWorker.run() line: 122	
	org.testng.TestRunner$$Lambda.0x00007fc93c0b63c8.accept(java.lang.Object) line: not available	
	java.util.ArrayList<E>.forEach(java.util.function.Consumer<? super E>) line: 1596	
	org.testng.TestRunner.privateRun(org.testng.xml.XmlTest) line: 829	
	org.testng.TestRunner.run() line: 602	
	org.testng.SuiteRunner.runTest(org.testng.TestRunner) line: 437	
	org.testng.SuiteRunner.runSequentially() line: 431	
	org.testng.SuiteRunner.privateRun() line: 391	
	org.testng.SuiteRunner.run() line: 330	
	org.testng.SuiteRunnerWorker.runSuite(org.testng.internal.invokers.SuiteRunnerMap, org.testng.xml.XmlSuite) line: 52	
	org.testng.SuiteRunnerWorker.run() line: 95	
	org.testng.TestNG.runSuitesSequentially(org.testng.xml.XmlSuite, org.testng.internal.invokers.SuiteRunnerMap, int, java.lang.String) line: 1256	
	org.testng.TestNG.runSuitesLocally() line: 1176	
	org.testng.TestNG.runSuites() line: 1099	
	org.testng.TestNG.run() line: 1067	
	org.apache.maven.surefire.testng.TestNGExecutor.run(java.lang.Iterable<java.lang.Class<?>>, java.lang.String, java.util.Map<java.lang.String,java.lang.String>, org.apache.maven.surefire.report.RunListener, java.io.File, org.apache.maven.surefire.testset.TestListResolver, java.util.List<org.apache.maven.surefire.cli.CommandLineOption>, int) line: 132	
	org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(org.apache.maven.surefire.report.RunListener, java.lang.Class<?>) line: 112	
	org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(org.apache.maven.surefire.util.TestsToRun, org.apache.maven.surefire.report.RunListener) line: 99	
	org.apache.maven.surefire.testng.TestNGProvider.invoke(java.lang.Object) line: 147	
	org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(java.lang.Object, java.lang.Object, org.apache.maven.surefire.booter.ProviderConfiguration, boolean, org.apache.maven.surefire.booter.StartupConfiguration, boolean) line: 290	
	org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(java.lang.Object, org.apache.maven.surefire.booter.StartupConfiguration, org.apache.maven.surefire.booter.ProviderConfiguration, java.io.PrintStream) line: 242	
	org.apache.maven.surefire.booter.ForkedBooter.main(java.lang.String...) line: 121	

```

But later it is set again and wrongly with value 123:
```
Thread [main] (Suspended (breakpoint at line 652 in io.smallrye.openapi.api.models.media.SchemaImpl))	
	owns: org.jboss.weld.Container  (id=155)	
	owns: org.eclipse.microprofile.openapi.tck.AirlinesAppTest  (id=156)	
	io.smallrye.openapi.api.models.media.SchemaImpl.setExample(java.lang.Object) line: 652	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c350400.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	io.smallrye.openapi.api.util.MergeUtil.mergeObjects(T, T) line: 133	
	io.smallrye.openapi.api.util.MergeUtil.mergeMaps(java.util.Map, java.util.Map) line: 179	
	io.smallrye.openapi.api.util.MergeUtil.mergeObjects(T, T) line: 115	
	io.smallrye.openapi.api.util.MergeUtil.mergeMaps(java.util.Map, java.util.Map) line: 179	
	io.smallrye.openapi.api.util.MergeUtil.mergeObjects(T, T) line: 115	
	io.smallrye.openapi.api.util.MergeUtil.mergeObjects(T, T) line: 104	
	io.smallrye.openapi.api.util.MergeUtil.merge(org.eclipse.microprofile.openapi.models.OpenAPI, org.eclipse.microprofile.openapi.models.OpenAPI) line: 57	
	io.smallrye.openapi.jaxrs.JaxRsAnnotationScanner.processApplicationClasses(io.smallrye.openapi.runtime.scanner.spi.AnnotationScannerContext, org.eclipse.microprofile.openapi.models.OpenAPI) line: 176	
	io.smallrye.openapi.jaxrs.JaxRsAnnotationScanner.scan(io.smallrye.openapi.runtime.scanner.spi.AnnotationScannerContext, org.eclipse.microprofile.openapi.models.OpenAPI) line: 151	
	io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.scan(java.lang.String...) line: 127	
	io.helidon.microprofile.openapi.MPOpenAPISupport(io.helidon.openapi.OpenAPISupport).lambda$expandModelUsingAnnotations$3(io.smallrye.openapi.api.OpenApiConfig, java.util.concurrent.atomic.AtomicReference, org.jboss.jandex.IndexView) line: 415	
	io.helidon.openapi.OpenAPISupport$$Lambda.0x00007fc93c505598.accept(java.lang.Object) line: not available	
	java.util.ArrayList<E>.forEach(java.util.function.Consumer<? super E>) line: 1596	
	io.helidon.microprofile.openapi.MPOpenAPISupport(io.helidon.openapi.OpenAPISupport).expandModelUsingAnnotations(io.smallrye.openapi.api.OpenApiConfig, java.util.List<? extends org.jboss.jandex.IndexView>) line: 412	
	io.helidon.microprofile.openapi.MPOpenAPISupport(io.helidon.openapi.OpenAPISupport).prepareModel(io.smallrye.openapi.api.OpenApiConfig, io.smallrye.openapi.runtime.OpenApiStaticFile, java.util.List<? extends org.jboss.jandex.IndexView>) line: 381	
	io.helidon.microprofile.openapi.MPOpenAPISupport(io.helidon.openapi.OpenAPISupport).lambda$model$0() line: 255	
	io.helidon.openapi.OpenAPISupport$$Lambda.0x00007fc93c4e6ad8.get() line: not available	
	io.helidon.openapi.OpenAPISupport.access(java.util.concurrent.locks.Lock, java.util.function.Supplier<T>) line: 1030	
	io.helidon.microprofile.openapi.MPOpenAPISupport(io.helidon.openapi.OpenAPISupport).model() line: 253	
	io.helidon.microprofile.openapi.MPOpenAPISupport(io.helidon.openapi.OpenAPISupport).prepareModel() line: 236	
	io.helidon.microprofile.openapi.MPOpenAPISupport.prepareModel() line: 32	
	io.helidon.microprofile.openapi.OpenApiCdiExtension.buildModel(java.lang.Object) line: 118	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c350400.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.jboss.weld.injection.VirtualMethodInjectionPoint<T,X>(org.jboss.weld.injection.StaticMethodInjectionPoint<T,X>).invoke(java.lang.Object, java.lang.Object[], java.lang.Class<? extends java.lang.RuntimeException>) line: 95	
	org.jboss.weld.injection.VirtualMethodInjectionPoint<T,X>(org.jboss.weld.injection.StaticMethodInjectionPoint<T,X>).invoke(java.lang.Object, java.lang.Object, org.jboss.weld.manager.BeanManagerImpl, jakarta.enterprise.context.spi.CreationalContext<?>, java.lang.Class<? extends java.lang.RuntimeException>) line: 85	
	org.jboss.weld.injection.MethodInvocationStrategy$SimpleMethodInvocationStrategy.invoke(java.lang.Object, org.jboss.weld.injection.MethodInjectionPoint<?,?>, T, org.jboss.weld.manager.BeanManagerImpl, jakarta.enterprise.context.spi.CreationalContext<?>) line: 168	
	org.jboss.weld.event.ExtensionObserverMethodImpl<T,X>(org.jboss.weld.event.ObserverMethodImpl<T,X>).sendEvent(T, java.lang.Object, jakarta.enterprise.context.spi.CreationalContext<?>) line: 330	
	org.jboss.weld.event.ExtensionObserverMethodImpl<T,X>.sendEvent(T, java.lang.Object, jakarta.enterprise.context.spi.CreationalContext<?>) line: 123	
	org.jboss.weld.event.ExtensionObserverMethodImpl<T,X>(org.jboss.weld.event.ObserverMethodImpl<T,X>).sendEvent(T) line: 308	
	org.jboss.weld.event.ExtensionObserverMethodImpl<T,X>(org.jboss.weld.event.ObserverMethodImpl<T,X>).notify(T) line: 286	
	org.jboss.weld.event.ExtensionObserverMethodImpl<T,X>(jakarta.enterprise.inject.spi.ObserverMethod<T>).notify(jakarta.enterprise.inject.spi.EventContext<T>) line: 125	
	org.jboss.weld.util.Observers.notify(jakarta.enterprise.inject.spi.ObserverMethod<? super T>, T, jakarta.enterprise.inject.spi.EventMetadata) line: 166	
	org.jboss.weld.event.ObserverNotifier.notifySyncObservers(java.util.List<jakarta.enterprise.inject.spi.ObserverMethod<? super T>>, T, jakarta.enterprise.inject.spi.EventMetadata, org.jboss.weld.event.ObserverNotifier.ObserverExceptionHandler) line: 285	
	org.jboss.weld.event.ObserverNotifier.notify(org.jboss.weld.event.ResolvedObservers<T>, T, jakarta.enterprise.inject.spi.EventMetadata) line: 273	
	org.jboss.weld.event.EventImpl<T>.fire(T) line: 96	
	io.helidon.microprofile.cdi.HelidonContainerImpl.doStart() line: 340	
	io.helidon.microprofile.cdi.HelidonContainerImpl$$Lambda.0x00007fc93c346f10.call() line: not available	
	io.helidon.common.context.Contexts.runInContext(io.helidon.common.context.Context, java.util.concurrent.Callable<T>) line: 137	
	io.helidon.microprofile.cdi.HelidonContainerImpl.start() line: 250	
	io.helidon.microprofile.server.ServerImpl.start() line: 76	
	io.helidon.microprofile.arquillian.ServerRunner.start(org.eclipse.microprofile.config.Config, int) line: 73	
	java.lang.invoke.LambdaForm$DMH.0x00007fc93c1a0000.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object, int) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c1a1800.invoke(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c1a1c00.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 155	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	io.helidon.microprofile.arquillian.HelidonDeployableContainer.startServer(io.helidon.microprofile.arquillian.HelidonDeployableContainer$RunContext, java.nio.file.Path[]) line: 318	
	io.helidon.microprofile.arquillian.HelidonDeployableContainer.deploy(org.jboss.shrinkwrap.api.Archive<?>) line: 196	
	org.jboss.arquillian.container.impl.client.container.ContainerDeployController$3.call() line: 151	
	org.jboss.arquillian.container.impl.client.container.ContainerDeployController$3.call() line: 118	
	org.jboss.arquillian.container.impl.client.container.ContainerDeployController.executeOperation(java.util.concurrent.Callable<java.lang.Void>) line: 239	
	org.jboss.arquillian.container.impl.client.container.ContainerDeployController.deploy(org.jboss.arquillian.container.spi.event.DeployDeployment) line: 118	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c110000.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.jboss.arquillian.core.impl.ObserverImpl.invoke(org.jboss.arquillian.core.spi.Manager, java.lang.Object) line: 86	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.invokeObservers() line: 103	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.proceed() line: 90	
	org.jboss.arquillian.container.impl.client.container.DeploymentExceptionHandler.verifyExpectedExceptionDuringDeploy(org.jboss.arquillian.core.spi.EventContext<org.jboss.arquillian.container.spi.event.DeployDeployment>) line: 47	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c110000.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.jboss.arquillian.core.impl.ObserverImpl.invoke(org.jboss.arquillian.core.spi.Manager, java.lang.Object) line: 86	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.proceed() line: 95	
	org.jboss.arquillian.container.impl.client.ContainerDeploymentContextHandler.createDeploymentContext(org.jboss.arquillian.core.spi.EventContext<org.jboss.arquillian.container.spi.event.DeploymentEvent>) line: 71	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c110000.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.jboss.arquillian.core.impl.ObserverImpl.invoke(org.jboss.arquillian.core.spi.Manager, java.lang.Object) line: 86	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.proceed() line: 95	
	org.jboss.arquillian.container.impl.client.ContainerDeploymentContextHandler.createContainerContext(org.jboss.arquillian.core.spi.EventContext<org.jboss.arquillian.container.spi.event.ContainerControlEvent>) line: 54	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c110000.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.jboss.arquillian.core.impl.ObserverImpl.invoke(org.jboss.arquillian.core.spi.Manager, java.lang.Object) line: 86	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.proceed() line: 95	
	org.jboss.arquillian.core.impl.ManagerImpl.fire(T, org.jboss.arquillian.core.spi.NonManagedObserver<T>) line: 133	
	org.jboss.arquillian.core.impl.ManagerImpl.fire(java.lang.Object) line: 105	
	org.jboss.arquillian.core.impl.EventImpl<T>.fire(T) line: 62	
	org.jboss.arquillian.container.impl.client.container.ContainerDeployController$1.perform(org.jboss.arquillian.container.spi.Container, org.jboss.arquillian.container.spi.client.deployment.Deployment) line: 92	
	org.jboss.arquillian.container.impl.client.container.ContainerDeployController$1.perform(java.lang.Object, java.lang.Object) line: 77	
	org.jboss.arquillian.container.impl.client.container.ContainerDeployController.forEachDeployment(java.util.List<org.jboss.arquillian.container.spi.client.deployment.Deployment>, org.jboss.arquillian.container.impl.client.container.ContainerDeployController.Operation<org.jboss.arquillian.container.spi.Container,org.jboss.arquillian.container.spi.client.deployment.Deployment>) line: 232	
	org.jboss.arquillian.container.impl.client.container.ContainerDeployController.forEachManagedDeployment(org.jboss.arquillian.container.impl.client.container.ContainerDeployController.Operation<org.jboss.arquillian.container.spi.Container,org.jboss.arquillian.container.spi.client.deployment.Deployment>) line: 212	
	org.jboss.arquillian.container.impl.client.container.ContainerDeployController.deployManaged(org.jboss.arquillian.container.spi.event.DeployManagedDeployments) line: 77	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c110000.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.jboss.arquillian.core.impl.ObserverImpl.invoke(org.jboss.arquillian.core.spi.Manager, java.lang.Object) line: 86	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.invokeObservers() line: 103	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.proceed() line: 90	
	org.jboss.arquillian.core.impl.ManagerImpl.fire(T, org.jboss.arquillian.core.spi.NonManagedObserver<T>) line: 133	
	org.jboss.arquillian.core.impl.ManagerImpl.fire(java.lang.Object) line: 105	
	org.jboss.arquillian.core.impl.EventImpl<T>.fire(T) line: 62	
	org.jboss.arquillian.container.test.impl.client.ContainerEventController.execute(org.jboss.arquillian.test.spi.event.suite.BeforeClass) line: 96	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c0bcc00.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.jboss.arquillian.core.impl.ObserverImpl.invoke(org.jboss.arquillian.core.spi.Manager, java.lang.Object) line: 86	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.invokeObservers() line: 103	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.proceed() line: 90	
	org.jboss.arquillian.test.impl.TestContextHandler.createClassContext(org.jboss.arquillian.core.spi.EventContext<org.jboss.arquillian.test.spi.event.suite.ClassEvent>) line: 83	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c110000.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.jboss.arquillian.core.impl.ObserverImpl.invoke(org.jboss.arquillian.core.spi.Manager, java.lang.Object) line: 86	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.proceed() line: 95	
	org.jboss.arquillian.test.impl.TestContextHandler.createSuiteContext(org.jboss.arquillian.core.spi.EventContext<org.jboss.arquillian.test.spi.event.suite.SuiteEvent>) line: 69	
	java.lang.invoke.DirectMethodHandle$Holder.invokeVirtual(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c110000.invoke(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 154	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.jboss.arquillian.core.impl.ObserverImpl.invoke(org.jboss.arquillian.core.spi.Manager, java.lang.Object) line: 86	
	org.jboss.arquillian.core.impl.EventContextImpl<T>.proceed() line: 95	
	org.jboss.arquillian.core.impl.ManagerImpl.fire(T, org.jboss.arquillian.core.spi.NonManagedObserver<T>) line: 133	
	org.jboss.arquillian.core.impl.ManagerImpl.fire(java.lang.Object) line: 105	
	org.jboss.arquillian.test.impl.EventTestRunnerAdaptor.beforeClass(java.lang.Class<?>, org.jboss.arquillian.test.spi.LifecycleMethodExecutor) line: 89	
	org.eclipse.microprofile.openapi.tck.AirlinesAppTest(org.jboss.arquillian.testng.Arquillian).arquillianBeforeClass() line: 99	
	java.lang.invoke.LambdaForm$DMH.0x00007fc93c0a8c00.invokeVirtual(java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.LambdaForm$MH.0x00007fc93c0a9400.invoke(java.lang.Object, java.lang.Object) line: not available	
	java.lang.invoke.Invokers$Holder.invokeExact_MT(java.lang.Object, java.lang.Object, java.lang.Object) line: not available	
	jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(java.lang.Object, java.lang.Object[]) line: 153	
	jdk.internal.reflect.DirectMethodHandleAccessor.invoke(java.lang.Object, java.lang.Object[]) line: 103	
	java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object...) line: 580	
	org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(java.lang.reflect.Method, java.lang.Object, java.lang.Object[]) line: 139	
	org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(org.testng.ITestNGMethod, org.testng.internal.ConstructorOrMethod, java.lang.Object, java.lang.Object[], org.testng.ITestResult) line: 69	
	org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(java.lang.Object, org.testng.ITestNGMethod, java.lang.Object[], org.testng.ITestResult) line: 361	
	org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(org.testng.internal.invokers.ConfigMethodArguments) line: 296	
	org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(org.testng.ITestClass, org.testng.IMethodInstance) line: 180	
	org.testng.internal.invokers.TestMethodWorker.run() line: 122	
	org.testng.TestRunner$$Lambda.0x00007fc93c0b63c8.accept(java.lang.Object) line: not available	
	java.util.ArrayList<E>.forEach(java.util.function.Consumer<? super E>) line: 1596	
	org.testng.TestRunner.privateRun(org.testng.xml.XmlTest) line: 829	
	org.testng.TestRunner.run() line: 602	
	org.testng.SuiteRunner.runTest(org.testng.TestRunner) line: 437	
	org.testng.SuiteRunner.runSequentially() line: 431	
	org.testng.SuiteRunner.privateRun() line: 391	
	org.testng.SuiteRunner.run() line: 330	
	org.testng.SuiteRunnerWorker.runSuite(org.testng.internal.invokers.SuiteRunnerMap, org.testng.xml.XmlSuite) line: 52	
	org.testng.SuiteRunnerWorker.run() line: 95	
	org.testng.TestNG.runSuitesSequentially(org.testng.xml.XmlSuite, org.testng.internal.invokers.SuiteRunnerMap, int, java.lang.String) line: 1256	
	org.testng.TestNG.runSuitesLocally() line: 1176	
	org.testng.TestNG.runSuites() line: 1099	
	org.testng.TestNG.run() line: 1067	
	org.apache.maven.surefire.testng.TestNGExecutor.run(java.lang.Iterable<java.lang.Class<?>>, java.lang.String, java.util.Map<java.lang.String,java.lang.String>, org.apache.maven.surefire.report.RunListener, java.io.File, org.apache.maven.surefire.testset.TestListResolver, java.util.List<org.apache.maven.surefire.cli.CommandLineOption>, int) line: 132	
	org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(org.apache.maven.surefire.report.RunListener, java.lang.Class<?>) line: 112	
	org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(org.apache.maven.surefire.util.TestsToRun, org.apache.maven.surefire.report.RunListener) line: 99	
	org.apache.maven.surefire.testng.TestNGProvider.invoke(java.lang.Object) line: 147	
	org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(java.lang.Object, java.lang.Object, org.apache.maven.surefire.booter.ProviderConfiguration, boolean, org.apache.maven.surefire.booter.StartupConfiguration, boolean) line: 290	
	org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(java.lang.Object, org.apache.maven.surefire.booter.StartupConfiguration, org.apache.maven.surefire.booter.ProviderConfiguration, java.io.PrintStream) line: 242	
	org.apache.maven.surefire.booter.ForkedBooter.main(java.lang.String...) line: 121	


```